tests/com/twitter/intellij/pants/settings/PantsProjectSettingsTest.java

```diff
@@ -0,0 +1,65 @@
+// Copyright 2017 Pants project contributors (see CONTRIBUTORS.md).
+// Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+package com.twitter.intellij.pants.settings;
+
+
+import com.intellij.ui.CheckBoxList;
+import com.intellij.util.containers.ContainerUtil;
+import com.twitter.intellij.pants.testFramework.OSSPantsIntegrationTest;
+
+import java.io.File;
+
+public class PantsProjectSettingsTest extends OSSPantsIntegrationTest {
+  public void testDirectory() {
```

Comment:
This first test is a little hard to follow.

How would you feel about breaking out some utility methods for the common components between it and testBuildFile. I'd also like to see some tests for the other states the control can be in, ie missing a file or non-pants project file.